website/docs/language/state/examples/encryption/fallback_from_unencrypted.tf

```diff
@@ -1,12 +1,17 @@
+locals {
+  # Change passphrase to be at least 16 characters long:
+  passphrase = "changeme!"
+}
+
 terraform {
   encryption {
     ## Step 1: Add the unencrypted method:
     method "unencrypted" "migrate" {}
 
     ## Step 2: Add the desired key provider:
     key_provider "pbkdf2" "mykey" {
-      # Change this to be at least 16 characters long:
-      passphrase = "changeme!"
+      ## You can also use variables here!

```

Comment:
Maybe we could change these examples to use variables instead of locals?

Reply:
Yep, changed back to var in [c821140](https://github.com/opentofu/opentofu/pull/1728/commits/c8211407d15e5356f837ef04376679389b56e3f0)